ci/release/update-version.sh

```diff
@@ -51,3 +51,10 @@ sed_runner 's/'"CUDA_DATAFRAME VERSION .* LANGUAGES"'/'"CUDA_DATAFRAME VERSION $
 # RTD update
 sed_runner 's/version = .*/version = '"'${NEXT_SHORT_TAG}'"'/g' docs/cudf/source/conf.py
 sed_runner 's/version = .*/version = '"'${NEXT_SHORT_TAG}'"'/g' docs/nvstrings/source/conf.py
+
+# bump rmm
+CURRENT_RMM="rmm=${CURRENT_MAJOR}.${CURRENT_MINOR}.*"
+NEXT_RMM="rmm=${NEXT_SHORT_TAG}.*"
```

Comment:
These variables should change.

`NEXT_SHORT_TAG` should be used instead of `NEXT_RMM`

`CURRENT_RMM` doesn't exist but we should redefine it near the top with the others as `CURRENT_SHORT_TAG`

`CURRENT_SHORT_TAG=${CURRENT_MAJOR}.${CURRENT_MINOR}`